src/content/en/2021/media.md

```diff
@@ -1,20 +1,875 @@
+        <td class="numeric">0.9%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 500px) 100vw, 500px`</td>
+        <td class="numeric">0.8%</td>
+        <td class="numeric">0.9%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 1024px) 100vw, 1024px`</td>
+        <td class="numeric">0.7%</td>
+        <td class="numeric">0.9%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 320px) 100vw, 320px`</td>
+        <td class="numeric">0.5%</td>
+        <td class="numeric">0.8%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 100px) 100vw, 100px`</td>
+        <td class="numeric">0.7%</td>
+        <td class="numeric">0.8%</td>
+      </tr>
+      <tr>
+        <td>`100vw`</td>
+        <td class="numeric">0.7%</td>
+        <td class="numeric">0.7%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="A ranked list of the most common sizes attribute values, before Javascript runs.", sheets_gid="228184279", sql_file="image_sizes_attribute_strings.sql") }}</figcaption>
+</figure>
+
+One in ten `sizes` attributes on mobile has an initial value of `auto`. This non-standard value is then presumably replaced by a Javascript library (probably <a hreflang="en" href="https://github.com/aFarkas/lazysizes">lazysizes.js</a>), using the measured layout size of the image.
+
+Some error in `sizes` is acceptable; the attribute provides a pre-layout hint to the browser in order to help it select an appropriate resource to load,_ before_ layout is complete. But large errors can lead to bad resource choices. This appears likely for the least-accurate quarter of `sizes` attributes, which report widths twice as large as the actual `<img>` layout width on desktop and 1.5x as large as the actual `<img>` layout width on mobile.
+
+So: one in ten `sizes` attributes is being authored on the client by a Javascript library, and at least one in four is inaccurate enough that the error is likely to impact resource selection. Both of these facts represent significant opportunities for either <a hreflang="en" href="https://github.com/ausi/respimagelint">existing tooling</a> or <a hreflang="en" href="https://github.com/whatwg/html/issues/4654">new web platform features</a> to help more authors get `sizes` right.
+
+##### `<picture>`
+
+The `<picture>` element serves a couple of use cases:
+
+1. Art direction, with the [`media` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#the_media_attribute)
+2. Type-switching, via the [`type` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#the_type_attribute)
+
+{{ figure_markup(
+  caption="The percentage of pages which use `<picture>`.",
+  content="6%",
+  classes="big-number",
+  sheets_gid="620965569",
+  sql_file="picture_distribution.sql"
+)
+}}
+
+`<picture>` is used much less frequently than `srcset`, reflecting its somewhat-niche pair of use-cases. Here's how usage breaks down between those two use cases:
```

Comment:
Oi! I like `<picture>` ;-)

But seriously those isn't format switching a non-niche case? especially with the number of new formats springing up and varying browser support?

Or do most hold off from that or handle it a different way?

Curious on your thoughts.

Reply:
You’re right that it's a feature with a lot of potential benefit, that's perhaps underutilized. Art direction, too (see the above aspect-ratio-on-mobile discussion).

I’ll re-work my language in order to say that there’s a lot of potential (but underutilized) utility, here. Rather than making it sound like there’s not much need. "Niche use case" isn't quite right.

A common alternative way to do format switching is with server-side content negotiation. I’ll work that in as well, (but have no data on how common it is — very hard to gather).

Reply:
Done